<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HastaEkle</class>
 <widget class="QDialog" name="HastaEkle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1" rowspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>2</width>
       <height>361</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <layout class="QFormLayout" name="formLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="lblTel">
       <property name="text">
        <string>&amp;Telefon No</string>
       </property>
       <property name="buddy">
        <cstring>leTel</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leTel"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblAdres">
       <property name="text">
        <string>&amp;Adres</string>
       </property>
       <property name="buddy">
        <cstring>teAdres</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="teAdres"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnKaydet">
       <property name="text">
        <string>Kaydet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnVazgec">
       <property name="text">
        <string>Vazgeç</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="3" rowspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblKronikHastaliklar">
       <property name="text">
        <string>&amp;Kronik Hastalıklar</string>
       </property>
       <property name="buddy">
        <cstring>leKronikHastaliklar</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLineEdit" name="leKronikHastaliklar"/>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QListWidget" name="lwAlerjiler"/>
     </item>
     <item row="7" column="0" colspan="3">
      <widget class="QListWidget" name="lwKronikHastaliklar"/>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="cbKanGrubu">
       <item>
        <property name="text">
         <string>A+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A−</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B−</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AB+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AB−</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0−</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="2">
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnKronikHastaliklarEkle">
         <property name="text">
          <string>Ekle</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnKronikHastaliklarSil">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Sil</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLineEdit" name="leAlerjiler"/>
     </item>
     <item row="4" column="2">
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnAlerjilerEkle">
         <property name="text">
          <string>Ekle</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnAlerjilerSil">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Sil</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblAlerjiler">
       <property name="text">
        <string>&amp;Alerjiler</string>
       </property>
       <property name="buddy">
        <cstring>leAlerjiler</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblKanGrubu">
       <property name="text">
        <string>&amp;Kan Grubu</string>
       </property>
       <property name="buddy">
        <cstring>cbKanGrubu</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblAdi">
       <property name="text">
        <string>&amp;Adı</string>
       </property>
       <property name="buddy">
        <cstring>leAdi</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leAdi"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblSoyadi">
       <property name="text">
        <string>&amp;Soyadı</string>
       </property>
       <property name="buddy">
        <cstring>leSoyadi</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leSoyadi"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblCinsiyet">
       <property name="text">
        <string>&amp;Cinsiyet</string>
       </property>
       <property name="buddy">
        <cstring>cbCinsiyet</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cbCinsiyet">
       <item>
        <property name="text">
         <string>ERKEK</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>KADIN</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblDogumTarihi">
       <property name="text">
        <string>&amp;Doğum Tarihi</string>
       </property>
       <property name="buddy">
        <cstring>deDogumTarihi</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="deDogumTarihi"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblKimlikNo">
       <property name="text">
        <string>&amp;Kimlik No</string>
       </property>
       <property name="buddy">
        <cstring>leKimlikNo</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="leKimlikNo"/>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leAdi</tabstop>
  <tabstop>leSoyadi</tabstop>
  <tabstop>cbCinsiyet</tabstop>
  <tabstop>deDogumTarihi</tabstop>
  <tabstop>leKimlikNo</tabstop>
  <tabstop>leTel</tabstop>
  <tabstop>teAdres</tabstop>
  <tabstop>cbKanGrubu</tabstop>
  <tabstop>leAlerjiler</tabstop>
  <tabstop>lwAlerjiler</tabstop>
  <tabstop>leKronikHastaliklar</tabstop>
  <tabstop>lwKronikHastaliklar</tabstop>
  <tabstop>btnVazgec</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnKaydet</sender>
   <signal>clicked()</signal>
   <receiver>HastaEkle</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnVazgec</sender>
   <signal>clicked()</signal>
   <receiver>HastaEkle</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leAlerjiler</sender>
   <signal>returnPressed()</signal>
   <receiver>btnAlerjilerEkle</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leKronikHastaliklar</sender>
   <signal>returnPressed()</signal>
   <receiver>btnKronikHastaliklarEkle</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
